samples/sensor/sht3xd/prj.conf

```diff
@@ -8,3 +8,11 @@ CONFIG_STDOUT_CONSOLE=y
 CONFIG_I2C=y
 CONFIG_SENSOR=y
 CONFIG_SHT3XD=y
+# If this one...
+#CONFIG_SHT3XD_TRIGGER=y
```

Comment:
Instead of adding commented out lines here, you can add another test(s) to sample.yaml and use the `extra_args` property. Should also add the new boards to the `platform_whitelist`

Reply:
Done.